jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -31,6 +31,14 @@
             description: 'sha1 or refname (e.g. origin/pr/N/head) to build'
     scm:
         - git:
+            url: https://github.com/kubernetes/test-infra
```

Comment:
pipelines are apparently very nice and would probably solve a lot of problems we have (e.g. rebuilding k8s like 5 times on PR Jenkins) but it's also untested territory...